### ISSUE TITLE
(RESOLVED) Can't playback above 480p without a lot of buffering 

### ISSUE THREAD

**Issue by willcaddy** [yuliskov/SmartTube#1778]
Hello there, I have noticed since one of the last few updates the playback can't handle going above 480p avc 0.87mbps. if I do I get lots of buffering as if the internet isn't able to keep up. However my internet is nearly 90mbps where I am and also I am able to watch 4k content just fine using the normal YouTube app. 

It's on a Sony Bravia Smart TV running Android TV. Any ideas? 

Thanks :)

**Comment by sieem** [comment]
I have the same issue, it looks like YouTube is throttling the connection. Youtube-dl has had similar issues in the past.

**Comment by harixxxx** [comment]
15.83 fixed the problem (video wait cursor spinning every few seconds) for me as well on firestick.
But with 15.83, going back 10 sec with 'left' click is sluggish.

**Comment by okibcn** [comment]
I have the same issue with a Google TV dongle. The same video works fine when switching to not logged user. I can't upgrade to 15.81 with the Google TV dongle ( see issue #1780 ) to check if that has been fixed.

**Comment by evenwebb** [comment]
Now fixed in the latest version! Thanks, devs!

This was driving me nuts.

**Comment by TheArgentinian** [comment]
Just started experiencing ridiculous amount of buffering no matter the quality. I tried the native YouTube TV app and there's no buffering.

**Comment by TheArgentinian** [comment]
> Now fixed in the latest version! Thanks, devs!
> 
> This was driving me nuts.

Last night was working fine for me but it's broke again.

**Comment by ezemraraxg** [comment]
yes Stable version is very buffering in all videos. used previous beta version too it was working fine before and now it's buffering too.

**Comment by willcaddy** [comment]
Thank you devs :) All fixed in the latest beta, didn't realise it had been released sorry!

**Comment by n3tman** [comment]
Updating from 15.62 Stable to [15.81](https://github.com/yuliskov/SmartTubeNext/tree/15.81) Stable fixed the issue for me (CIS region). Thanks so much!

**Comment by Chealer** [comment]
Thank you @willcaddy 
Would you mind clarifying that this issue is resolved, perhaps by adding "RESOLVED" at the start of the ticket’s title?

**Comment by willcaddy** [comment]
Of course :) My apologies I thought I had done it!
